docs/extensions-integrations/integration-guides/calling-commands.md

```diff
@@ -25,13 +95,71 @@ public class ExampleIntegration  {
         // ... setup configuration system and file access
 
         try {
-            new CommandScope("update")
-                    .addArgumentValue("changeLogFile", "my/changelog.xml")
-                    .execute();
+            CommandScope updateCommand = new CommandScope("update");
+            updateCommand.addArgumentValue(UpdateCommandStep.CHANGELOG_FILE_ARG, changeLogFile);
+            updateCommand.addArgumentValue(UpdateCommandStep.CONTEXTS_ARG, contexts);
+            updateCommand.addArgumentValue(UpdateCommandStep.LABEL_FILTER_ARG, labelFilter);
+            updateCommand.addArgumentValue(ChangeExecListenerCommandStep.CHANGE_EXEC_LISTENER_CLASS_ARG, changeExecListenerClass);
+            updateCommand.addArgumentValue(ChangeExecListenerCommandStep.CHANGE_EXEC_LISTENER_PROPERTIES_FILE_ARG, changeExecListenerPropertiesFile);
+            setDatabaseArgumentsToCommand(updateCommand);
+            updateCommand.execute();
         } catch (CommandExecutionException e) {
             System.out.println("Error running update: "+e.getMessage());
         }
+    }
+}
+```
+
+### Execution in scope example
+
+```java
+package com.example;
+
+import liquibase.command.CommandScope;
+import liquibase.exception.CommandExecutionException;
+
+public class ExampleIntegration  {
 
+    public static void main(String[] args) throws Exception {
+        // ... setup configuration system and file access
+
+        try {
+            runInScope(() -> {
```

Comment:
the runInScope method is from liquibase class.. I think you need to use the scope.child here....